module_16/tests/demoblaze_test.robot

```diff
@@ -0,0 +1,22 @@
+*** Settings ***
+Library  SeleniumLibrary
+Resource  ../resources/resources.robot
+Test Setup  Open Browser    ${url}    ${browser}
+
+*** Test Cases ***
+LoginTest
+    ClickLogInButton
+    ValidateLoginAndPasswordFieldsArePresented
+    LoginToApplication
+    CheckLogOutButtonIsVisible
+    ValidateWelcomeMessageText  Welcome ${login}
+    close browser
```

Comment:
closing browser is not a step of scenario, as an environment cleaning it should be moved to Test Teardown

Reply:
Thanks! Fixed.